TimeService.test.ts

```diff
@@ -0,0 +1,21 @@
+import {TimeService} from "./TimeService";
+
+describe('TimeService', () => {
+    it('can test getCurrentTimeString', () => {
+        const now = new Date();
+        const currentTimeString = now.toISOString();
+
+        expect(TimeService.getCurrentTimeString()).toBeTruthy();
+        expect(parseInt(TimeService.getCurrentTimeString())).toBeGreaterThanOrEqual(parseInt(currentTimeString));
+        expect.assertions(2)
+
+    });
+
+    it('can test getTimeAfterMonths', () => {
+        const now = new Date();
+        const currentTimeString = now.toISOString();
+
+        expect(TimeService.getTimeAfterMonths(currentTimeString, 12)).toBeTruthy();
+
```

Comment:
You could add another tests here too, check what the expect prints out by using toBe('') for example, as tests tell you the outcome of the test.

You could add test case for atleast months 3 & 8 and something that is not supposed to be valid like 14